protect-function-app/.gitignore

```diff
@@ -0,0 +1,43 @@
+bin
+obj
+csx
+.vs
+edge
+Publish
+
+*.user
+*.suo
+*.cscfg
+*.Cache
+project.lock.json
+
+/packages
+/TestResults
+
+/tools/NuGet.exe
+/App_Data
+/secrets
+/data
+.secrets
+appsettings.json
+local.settings.json
+
+node_modules
+dist
+
+# Local python packages
+.python_packages/
+
+# Python Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
```

Comment:
where this this come from?  Might want to make sure you're using a node-specific version here instead of what looks like a python version?

Reply:
I was caught off-guard by this as well but it seems all languages have the same gitignore file. I even started from scratch and tried it again and it was the same...

![image](https://user-images.githubusercontent.com/82831332/151246692-38702f1f-95e0-48f3-b189-5a2ffeb4d4ef.png)

Reply:
It came from
https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local

Reply:
TIL!  I can confirm that is the output of `func new` -- Marking resolved.  Sorry for the noise, but good to know